workspaces/optic-streams/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../config/tsconfig.json",
   "compilerOptions": {
-    "outDir": "lib",
+    "outDir": "build",
```

Comment:
yay

Reply:
"lib" used to be where node packages had their JS code (generated or not), but TS seems to have muddled the water, so might as well be consistent with the rest of our workspaces 😅